app/screens/channel_info/index.js

```diff
@@ -100,8 +105,20 @@ function mapStateToProps(state) {
         timeZone = getUserCurrentTimezone(currentUser.timezone);
     }
 
+    let canUndeleteChannel = false;
+    if (hasNewPermissions(state)) {
```

Comment:
```suggestion
    if (hasNewPermissions(state) && isMinimumServerVersion(serverVersion, 5, 20)) {
```

it is OK to leave it as 5.18 for now for testing purposes, but the PR should not be merged until this is changed for 5.20 or 6.0 if the Feb release number changes

Reply:
what version should i be now?